core/camel-core-model/src/main/java/org/apache/camel/model/MarshalDefinition.java

```diff
@@ -88,6 +89,7 @@ public class MarshalDefinition extends NoOutputDefinition<MarshalDefinition> imp
             @XmlElement(name = "custom", type = CustomDataFormat.class),
             @XmlElement(name = "fhirJson", type = FhirJsonDataFormat.class),
             @XmlElement(name = "fhirXml", type = FhirXmlDataFormat.class),
+            @XmlElement(name = "fury", type = FuryDataFormat.class),
```

Comment:
You should add this in other places, just grep for one of the existing ie in Unmarshal and DataFormats definition classes

Reply:
Thanks @davsclaus for pointing out. Just wonder if there is any docs for develop a new `DataFormat`?

Reply:
No specific docs - the best is looking at any of the others and see how they are done.